editor/header/preview-button/index.js

```diff
@@ -73,6 +73,16 @@ export class PreviewButton extends Component {
 			'about:blank',
 			this.getWindowTarget()
 		);
+
+		const popupLoader = renderToString(
```

Comment:
Honestly seems like we could get away with just creating a string of HTML instead of using React to convert an element to string, particularly if we're separately creating a style tag anyways (which could itself be represented as an element).

Reply:
Good shout - I was only doing this because I'd originally used the `<Spinner />` component. No longer necessary 🙂